data.sql

```diff
@@ -19,4 +19,35 @@ values ('Angemon', DATE '2005-06-12', 1, true, -45);
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
 values ('Boarmon', DATE '2005-06-07', 7, true, 20.4);
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
-values ('Blossom', DATE '1998-10-13', 3, true, 17);
\ No newline at end of file
+values ('Blossom', DATE '1998-10-13', 3, true, 17);
+
+INSERT INTO owners(full_name, age)
+values ('Sam Smith', 34);
+INSERT INTO owners(full_name, age)
+values ('Jennifer Orwell', 19);
+INSERT INTO owners(full_name, age)
+values ('Bob', 45);
+INSERT INTO owners(full_name, age)
+values ('Melody Pond', 77);
+INSERT INTO owners(full_name, age)
+values ('Dean Winchester', 14);
+INSERT INTO owners(full_name, age)
+values ('Jodie Whittaker', 38);
+
+INSERT INTO species(name)
+values ('Pokemon');
+INSERT INTO species(name)
+values ('Digimon');
+
+BEGIN;
+UPDATE animals SET species_id = 2 WHERE name like '%mon';
+UPDATE animals SET species_id = 1 WHERE species_id is null;
+COMMIT;
+
+BEGIN;
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 2 WHERE name = 'Gabumon' or name = 'Pikachu';
+UPDATE animals SET owner_id = 3 WHERE name = 'Devimon' or name = 'Plantmon';
+UPDATE animals SET owner_id = 4 WHERE name = 'Charmander' or name = 'Squirtle' or name = 'Blossom';
+UPDATE animals SET owner_id = 5 WHERE name = 'Angemon' or name = 'Boarmon';
+COMMIT;
```

Comment:
- [ ]Again,  Instead of setting the value using IDs, let us use select id from owners that is match with condition.